packages/markdown/remark/src/highlight.ts

```diff
@@ -21,7 +23,11 @@ const languagePattern = /\blanguage-(\S+)\b/;
  *   A function which receives the code and language, and returns the HTML of a syntax
  *   highlighted `<pre>` element.
  */
-export async function highlightCodeBlocks(tree: Root, highlighter: Highlighter) {
+export async function highlightCodeBlocks(
+	tree: Root,
+	highlighter: Highlighter,
+	{ excludeLanguages = defaultExcludeLanguages }: { excludeLanguages?: string[] } = {},
```

Comment:
Why do we pass a third argument as an object instead of a plain `excludeLanguages`? What does the new argument represent? Options or something?

Reply:
Changed it to a simple argument.